WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -223,9 +228,17 @@ public void onResume() {
             mFilterRadioGroup.setVisibility(View.GONE);
         } else {
             getNotesAdapter().reloadNotesFromDBAsync();
+            if (mShouldRefreshNotifications) {
+                fetchNotesFromRemote();
+            }
         }
     }
 
+    @Override public void onPause() {
```

Comment:
Let's add a newline here, after `@Override`

Reply:
Well that's embarrassing... and not a nitpick at all! I'd hope you'd flag this everytime you saw it.

Thanks for catching this!